### PR TITLE
Include update severity level in updateinfo.xml

### PR DESCRIPTION
**Pull request by jlebon**
Right now, doing a `dnf list-sec` shows e.g.:

```
FEDORA-2017-fb5e227432 Unknown/Sec. glibc-2.26-20.fc27.x86_64
```

The `Unknown` in the output is due to the fact that `updateinfo.xml`
doesn't seem to include the severity level of the security update even
though the update does have it set (to "low" in this case). RHEL
repodata on the other hand does include this, and it shows up in `yum
list-sec`.

This is an important factor for users when determining when to apply the
updates. So let's include it in the XML.

Signed-off-by: Jonathan Lebon <jlebon@redhat.com>

### CODE REVIEW

**Comment by jlebon** [comment]
Let me know if I forgot to update (pun intended) something somewhere!

I've also verified on the client side that `dnf list-sec` picks this up and displays it correctly:

```
$ sudo dnf  --disablerepo='*' --enablerepo=bodhitest --enablerepo=bodhitest2 list-sec all
Repository 'bodhitest' is missing name in configuration, using id.
Repository 'bodhitest2' is missing name in configuration, using id.
Repository 'fahc' is missing name in configuration, using id.
bodhitest                                                                              966 kB/s | 989  B     00:00
Last metadata expiration check: 0:00:00 ago on Thu 21 Dec 2017 08:15:12 PM UTC.
  FEDORA-2017-ad1937a146 Moderate/Sec. TurboGears-1.0.2.2-2.fc17.noarch
```

We'll be making use of this in rpm-ostree as well in the context of [automatic updates](https://github.com/projectatomic/rpm-ostree/issues/247).

**Comment by jlebon** [comment]
📞 Can someone have a look at this? Thanks!

File: README.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ Fedora system and pointing your browser at
 If you are interested in contributing to Bodhi, you can read the
 `developer documentation`_.
 
-.. _developer documentation: https://bodhi.fedoraproject.org/docs/developer/index.html
+.. _developer documentation: https://bodhi.fedoraproject.org/docs/developer_docs.html
```
Comment (bowlofeggs) [inline README.rst]: This change is unrelated, but is also actually a regression - the upcoming 3.2.0 release will make the existing link correct once it is deployed to production.

**Review by bowlofeggs — CHANGES_REQUESTED** [review]
Just one requested change, looks good otherwise!

Also sorry for the delay in looking at this - I've been AFK for more than three weeks!

**Comment by bowlofeggs** [comment]
jenkies test

**Comment by jlebon** [comment]
Thanks, dropped that commit! ⬆️

**Review by bowlofeggs — APPROVED** [review]

**Comment by bowlofeggs** [comment]
jenkies test

**Comment by jlebon** [comment]
Fixed flake8 failure! ⬆️

**Comment by bowlofeggs** [comment]
I just rebased your branch to fix a merge conflict. Once the CI tests pass (fingers crossed) we'll be good to merge!

**Comment by bowlofeggs** [comment]
jenkies test

**Comment by bowlofeggs** [comment]
Ah, simple docstyle fix - I'll take care of it for you!

**Comment by bowlofeggs** [comment]
jenkies test

**Comment by bowlofeggs** [comment]
The patch for this feature has been included in the [```3.3.0b0``` build](https://bodhi.stg.fedoraproject.org/docs/user/release_notes.html), which is currently deployed to [staging](https://bodhi.stg.fedoraproject.org).

**Pull request merged**